cypress/tests/ImageElement.spec.ts

```diff
@@ -211,6 +218,12 @@ describe("ImageElement", () => {
         typeIntoElementField("src", "Src text");
         assertDocHtml(getSerialisedHtml({ srcValue: "Src text" }));
       });
+
+      it("should deserialise content from HTML into the appropriate node in the document", () => {
```

Comment:
Do we want tests like these for other field types i.e. checkbox, dropdown?

Reply:
I think it'd be good to have them in a separate PR – I've added a card to our backlog.